spec/rails/doc/smart/expected.yaml

```diff
@@ -28,16 +28,19 @@ paths:
         example: token
       - name: page
         in: query
+        required: false
         schema:
           type: integer
         example: 42
       - name: per
         in: query
+        required: false
         schema:
           type: integer
         example: 10
       - name: show_columns
         in: query
+        required: false
```

Comment:
@Alexey2257 Could you update one of the test case to include `required_request_params: [ 'show_columns']` so we ensure `required: true` is generated?

Reply:
Sure, done